File: python/ubervan.py
from carcustomized import CarCustomized

class UberVan(CarCustomized):
    
    def __init__(self, id,number_plate, category, assembly_date, enrolled, brand, model, enrrollment_date, passengers, capacity, driver, seat_material, type_car_accepted):
        super.__init__(id,number_plate, category, assembly_date, enrolled, brand, model, enrrollment_date, passengers, capacity, driver, seat_material, type_car_accepted)

File: java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends CarCustomized
{
    public UberVan(String number_plate, Account driver, Map<String, Map<String,Integer>> type_car_accepted, ArrayList<String> seat_material)
    {
        super(number_plate, driver, type_car_accepted, seat_material);
    }
    
    @Override
    public void setCapacity(Integer capacity) {
        if(capacity == 6)
        {
            super.capacity=capacity;            
        }
        else
        {
            System.out.println("Please insert a valid capacity");
        }        
    }
}
